install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -175,6 +175,14 @@ data:
         - name: ISTIO_META_NETWORK
           value: "{{ .Values.global.network }}"
         {{- end }}
+        - name: ISTIO_META_WORKLOAD_SDS_ENABLED
```

Comment:
Also: to avoid another 'this can't be tested unless you set global setting' - please also add an if on a pod annotation, so we don't have to do it again and can be tested on regular clusters, at least this part. 

Pod annotations can be easily checked in the template, I think we have a few.

Reply:
done.